src/schema/v2/artworksByInternalID.ts

```diff
@@ -0,0 +1,21 @@
+import Artwork from "./artwork"
+import {
+  GraphQLList,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+export const ArtworksByInternalID: GraphQLFieldConfig<void, ResolverContext> = {
+  type: new GraphQLList(Artwork.type),
```

Comment:
#minor this can prob be a `new GraphQLNonNull` as well.

Reply:
Ah 👍 will update